### PR TITLE
Add change feed properties to 2019-04-01 Storage.

### PR DESCRIPTION
**Pull request by tonykjose**
Add change feed properties to Storage Blob Services.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
Is this a readonly property, or not?

If not, it might break GET-PUT pipeline. https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?rtime=oPAFQY8Z10g

**Comment by tonykjose** [comment]
It is not a read-only property. Having said this, unless the customer explicitly disables the property by explicitly setting the property to false, our PUT semantics will ignore the update on the property. This is consistent with all Storage APIs i.e. PUT as update = PATCH and not a full resource replace.

-Tony

**Comment by weidongxu-microsoft** [comment]
@tonykjose So if user1 with latest SDK call PUT and set the property to **false**, and then user2 with old SDK call GET, change some other property, then PUT without this property, it will stay **false**. Then I think it should be fine.

I think I still need ARM signoff.

**Comment by tonykjose** [comment]
Yes that is correct.

-Tony
________________________________

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json [hunk 1]
```diff
@@ -1170,6 +1174,15 @@
         }
       }
     },
+    "ChangeFeed": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
```
Comment (KrisBash) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json]: booleans are less preferred as it becomes a breaking change if you need to change to an enum later. If you model this as something like '"state":[enabled|disabled]' you can add tertiary states to the enum later, without a breaking change. Something to consider.

**Review by KrisBash — APPROVED** [review]
Please see one comment, but signing off

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**